doc/sphinx-guides/source/admin/troubleshooting.rst

```diff
@@ -98,14 +98,12 @@ We don't know what's causing this issue, but here's a known workaround:
 
 - Stop Payara; 
 
-- Remove the ``generated`` and ``osgi-cache`` directories; 
-
-- Delete all the rows from the ``EJB__TIMER__TBL`` table in the database;
+- Remove the ``generated`` and ``osgi-cache`` directories;
 
 - Start Payara
 
 The shell script below performs the steps above. 
-Note that it may or may not work on your system, so it is provided as an example only, downloadable :download:`here </_static/util/clear_timer.sh>`. Aside from the configuration values that need to be changed to reflect your environment (the Payara directory, name of the database, etc.) the script relies on the database being configured in a certain way for access. (See the comments in the script for more information)
+Note that it may or may not work on your system, so it is provided as an example only, downloadable :download:`here </_static/util/clear_timer.sh>`. The configuration values to reflect your environment (the Payara directory) might need to be changed. (See the comments in the script for more information)
```

Comment:
very minor comment, but I think this would read better the way it was before (outside of the removing the aside): " The configuration values might need to be changed to reflect your environment (the Payara directory)."

Reply:
I went ahead and made the change myself.